carto/kuvizs.py

```diff
@@ -42,6 +43,28 @@ class Meta:
         collection_endpoint = API_ENDPOINT.format(api_version=API_VERSION)
         name_field = "id"
 
+    def update(self, html, name, password):
```

Comment:
Now that I read the code again... We don't want the API to be like this.

As per Managers and Resources work, if we want to update a Resource we should do:

```python
kuviz = KuvizManager.get(name) # or id, depending how the REST API works
kuviz.name = ...
kuviz.html = ...
kuviz.password = ...
kuviz.save()
```

Reply:
There are also a couple of custom fields here.

The `html` could be a `Base64EncodedField` wich should have the encoding logic included.
The `password` one should have the logic for changing the privacy as well.

Reply:
About the manager in the update: we already have the resource, IMO it has no sense to make a new request to have what you already have. Should we always use the manager instead of the resource?

Reply:
It was an example... what I mean is to be consistent within the SDK: Managers are for creating / listing / filtering. Resources are for saving / deleting.

Not saying is the best design, but it's what we have and it makes easier for us to explain how to deal with our APIs via the SDK (there are exceptions as the SQLClient which is not a RESTful API).

Maybe, if you don't want to get the resource first, just initializing it migth work (if the name is used as the id in the REST API)

```python
kuviz = Kuviz.init(name='...', html='...', etc.)
kuviz.save()
```

In any case, what it's important here (in terms of implementation) is that setting an attribute and then saving should work:

 ```python
resource.html = the_html
resource.save()
```